src/main/java/org/example/Exceptions/Entity.java

```diff
@@ -0,0 +1,21 @@
+package org.example.Exceptions;
+
+public enum Entity {
```

Comment:
should this be a UserRole enum?

Reply:
Deleted file as the enum ended up not being used.